fusedrug/utils/cluster/cluster_using_mmseqs.py

```diff
@@ -111,14 +116,17 @@ def cluster(
             cmd = f"gunzip {input_fasta_filename}.gz"
             _run_system_cmd(cmd)
 
-    os.makedirs(join(output_dir, "mmseqs_workspace"))
+    # Create workspace (supports overriding)
+    workspace_dir = join(output_dir, "mmseqs_workspace")
+    if override_workspace and os.path.exists(workspace_dir):
+        shutil.rmtree(workspace_dir)
 
+    os.makedirs(workspace_dir)
```

Comment:
great - I like this (not setting exist_ok=True) - as we don't want to clutter and make people unintentionally delete directories with other things.